Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -40,10 +40,8 @@ WbfsFileReader::WbfsFileReader(const std::string& filename)
 
 WbfsFileReader::~WbfsFileReader()
 {
-	for (u32 i = 0; i != m_files.size(); ++ i)
-	{
-		delete m_files[i];
-	}
+	for (auto& f : m_files[i])
```

Comment:
It's pretty much a perfect use case for auto here — nothing depends on the exact type in that loop.